### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by Mauwa28** [community/community#50063] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi There, 
I am a novice in the field of data analysis, and I am currently engaged in a project involving SQL and utilizing Microsoft SQL Server Management Studio 2019. Regrettably, while attempting to import an Excel file, an error message has been received, indicating that the "Microsoft.ACE.OLEDB.12.0" provider has not been registered on the local machine. Kindly provide me with assistance in resolving this matter.
Thank you!

**Comment by jricciardi** [comment]
Hi there! The `Discussions` category is an area about the product, GitHub Discussions. I've moved this to the more appropriate `General` category.

**Reply by Mauwa28** [reply]
Oh I am so sorry about that.
Thank you for your help! :)
